tests/components/startca/test_sensor.py

```diff
@@ -53,51 +58,51 @@ async def test_capped_setup(hass, aioclient_mock):
     await hass.async_block_till_done()
 
     state = hass.states.get("sensor.start_ca_usage_ratio")
-    assert state.attributes.get("unit_of_measurement") == PERCENTAGE
+    assert state.attributes.get(ATTR_UNIT_OF_MEASUREMENT) == PERCENTAGE
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)